becquerel/core/rebin.py

```diff
@@ -366,12 +273,12 @@ def rebin(in_spectra, in_edges, out_edges, method="interpolation",
     Spectra rebinning via deterministic or stochastic methods.
 
     Args:
-        in_spectrum (np.ndarray): an array of input spectrum counts (1D or 2D)
-            [num_spectra, num_bins_in] or [num_bins_in]
-        in_edges (np.ndarray): an array of the input bin edges (1D or 2D)
-            [num_spectra, num_bins_in + 1] or [num_bins_in + 1]
+        in_spectrum (np.ndarray): an array of input spectrum counts
+            [..., num_bins_in]
+        in_edges (np.ndarray): an array of the input bin edges
+            [..., num_bins_in + 1] or [num_bins_in + 1]
```

Comment:
Add more detail for the two cases (i.e. all the same edges, or different edges matching the number of spectra in)

Reply:
i.e., Explain the two cases more explicitly so that someone can more immediately understand what they mean.

Reply:
Just fixed this hope it is good now.